db/migrate/20220722225402_create_account_debits.rb

```diff
@@ -0,0 +1,14 @@
+class CreateAccountDebits < ActiveRecord::Migration[6.0]
+  def change
+    create_table :account_debits do |t|
+      t.string :bank, null: false
+      t.string :branch
+      t.string :number, null: false
+      t.decimal :credit_limit, precision: 10, scale: 2, null: false, default: 0
+      t.string :owner_name
+      t.references :account, null: false, foreign_key: true
+      t.timestamps
+    end
+    add_reference :account_debits, :owner, foreign_key: { to_table: :users }, null: false
```

Comment:
Same as [here](https://github.com/wfbraga/finco/pull/15/files#r961631084).

Reply:
_Important note:_ Remember to rollback this migration before editting it.

With `rails db:migrate:down VERSION=20220722225402`